src/auth_client_demo_assets/svelte/src/routes/+page.svelte

```diff
@@ -0,0 +1,21 @@
+<script>
+  import LoggedIn from "../components/LoggedIn.svelte";
+  import LoggedOut from "../components/LoggedOut.svelte";
+  import { auth } from "../stores/auth";
+  import { onMount } from "svelte";
+
+  onMount(() => {
+    window.global = window;
```

Comment:
any reason not to polyfill global in the `vite.config`?

Reply:
it breaks during sveltekit SSR. If you know how to do the polyfill correctly I'd love to know!

Reply:
Try this: https://github.com/buildwithjuno/juno/blob/e40ad9f842a724af1b3e925a18312314f3010dad/vite.config.ts#LL110C7-L110C7

Reply:
Thanks, works great!